pombola/core/static/js/hover-dropdown.js

```diff
@@ -0,0 +1,69 @@
+$(function(){
+
+    /*
```

Comment:
I wonder whether, if this comment were in single line comment format (`//`) whether it'd get stripped out by the django pipeline? Seems a shame to be sending all this documentation to the client.